cypress/e2e/tests/pages/manager/gke-cluster-provisioning.spec.ts

```diff
@@ -0,0 +1,112 @@
+import LoadingPo from '@/cypress/e2e/po/components/loading.po';
+import ClusterManagerCreateGKEPagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-gke.po';
+import { DEFAULT_GCP_ZONE } from '@/pkg/gke/util/gcp';
+
+// will only run this in jenkins pipeline where cloud credentials are stored
+describe('Deploy GKE cluster with default settings', { tags: ['@manager', '@adminUser', '@standardUser', '@jenkins'] }, () => {
+  const clusterList = new ClusterManagerListPagePo();
+  const loadingPo = new LoadingPo('.loading-indicator');
+
+  let cloudcredentialId = '';
+  const gkeDefaultZone = 'us-central1-c';
+  let gkeVersion = '';
+  // clusterId commented out as it will be used for the next tests of this suite, but not being used by the current test
+  // let clusterId = '';
+  let clusterDescription = '';
+  const gkeProjectId = JSON.parse(Cypress.env('gkeServiceAccount')).project_id;
+
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+
+    // clean up GKE cloud credentials
+    cy.getRancherResource('v3', 'cloudcredentials', null, null).then((resp: Cypress.Response<any>) => {
+      const body = resp.body;
+
+      if (body.pagination['total'] > 0) {
+        body.data.forEach((item: any) => {
+          if (item.googlecredentialConfig) {
+            const id = item.id;
+
+            cy.deleteRancherResource('v3', 'cloudcredentials', id);
+          } else {
+            cy.log('There are no existing GKE cloud credentials to delete');
+          }
+        });
+      }
+    });
+  });
+
+  beforeEach(() => {
+    cy.createE2EResourceName('gkecluster').as('gkeClusterName');
+    cy.createE2EResourceName('gkecloudcredential').as('gkeCloudCredentialName');
+  });
+
+  it('Successfully create GKE cluster with default settings', function() {
+    const createGKEClusterPage = new ClusterManagerCreateGKEPagePo();
+    const cloudCredForm = createGKEClusterPage.cloudCredentialsForm();
+
+    // Select GKE and create GKE cluster page
+    ClusterManagerListPagePo.navTo();
+    clusterList.waitForPage();
+    clusterList.createCluster();
+    createGKEClusterPage.selectKubeProvider(2);
+    loadingPo.checkNotExists();
+    createGKEClusterPage.rke2PageTitle().should('include', 'Create Google GKE');
+    createGKEClusterPage.waitForPage('type=googlegke&rkeType=rke2');
+
+    // create GKE cloud credential
+    cloudCredForm.saveButton().expectToBeDisabled();
+    cloudCredForm.nameNsDescription().name().set(this.gkeCloudCredentialName);
+    cloudCredForm.serviceAccount().set(Cypress.env('gkeServiceAccount'));
+    cloudCredForm.saveButton().expectToBeEnabled();
+    cy.intercept('GET', '/v1/management.cattle.io.users?exclude=metadata.managedFields').as('pageLoad');
+    cloudCredForm.saveCreateForm().cruResource().saveAndWaitForRequests('POST', '/v3/cloudcredentials').then((req) => {
+      expect(req.response?.statusCode).to.equal(201);
+      cloudcredentialId = req.response?.body.id.replace(':', '%3A');
```

Comment:
This should be removed post test

Reply:
I'm not sure we would gain much by that, given the test could just be run again with additional info / debug output and recordings of the test should also show the issue.

This does raise a question though, what deletes the created cluster?